### ISSUE TITLE
sphinx >= 7.2 appears to break `piccolo` theme

### ISSUE THREAD

**Issue by alexlancaster** [piccolo-orm/piccolo_theme#61]
I upgraded to sphinx 7.2.2 and I now get this error (also happens with 7.2.0, 7.2.1 - seems to have been introduced in 7.2.x series):

```
writing output... [ 10%] docs/AUTHORS
Theme error:
An error happened in rendering the page docs/AUTHORS.
Reason: ThemeError("Local asset file paths must not contain query strings: '_static/basic_mod.css?v=0.7.0-1'")
```
If I revert to 7.1.2 and earlier, I don't encounter this error.

**Comment by alexlancaster** [comment]
I wonder if this problem is related to this: https://github.com/sphinx-doc/sphinx/issues/11608 ?

**Comment by dantownsend** [comment]
@alexlancaster Thanks for reporting this.

Seems like there was a lot of changes in Sphinx 7.2

The reason we add a GET param to the CSS path is for cache busting - browsers will ignore their cached CSS if they see a different GET param.

If we remove it get GET param, it's not the end of the world.

**Comment by dantownsend** [comment]
I just checked, and removing the GET param fixes it - will do a release tomorrow.

**Comment by dantownsend** [comment]
This should be fixed now in `piccolo_theme==0.17.0`. 

I've added some integration tests, so we're now testing against a range of Sphinx versions.

**Comment by alexlancaster** [comment]
I just `pip` updated to latest `piccolo-theme` and `sphinx` 7.2.2 and my docs build fine now.  Thank you!
